docs/resources/projects/settings/project-setup.md

```diff
@@ -532,4 +532,127 @@ To resolve this, simply add a delay ([Wait](../../../resources/control-flow/time
+- **Dev Environment**: Use for testing and developing new features without affecting production data.
+- **Staging Environment**: Use to simulate the production environment before launching, and is isolated from the actual production data.
+
+
+*This is a common best practice, but you can create custom environments with different names for your own workflow style.*
+:::
+
+### Create and Switch Environments
+
+To create and switch environments, follow the steps below:
+
+<div style={{
+    position: 'relative',
+    paddingBottom: 'calc(56.67989417989418% + 41px)', // Keeps the aspect ratio and additional padding
+    height: 0,
+    width: '100%'}}>
+    <iframe 
+        src="https://demo.arcade.software/yR8P5pFPOKtuQ0jFSOJ7?embed&show_copy_link=true"
+        title=""
+        style={{
+            position: 'absolute',
+            top: 0,
+            left: 0,
+            width: '100%',
+            height: '100%',
+            colorScheme: 'light'
+        }}
+        frameborder="0"
+        loading="lazy"
+        webkitAllowFullScreen
+        mozAllowFullScreen
+        allowFullScreen
+        allow="clipboard-write">
+    </iframe>
+</div>
+<p></p>
+
+### Use Environment Variables
+Let's see an example of creating and using `apiUrl` as an environment variable in an e-commerce app.
+
+<div style={{
+    position: 'relative',
+    paddingBottom: 'calc(56.67989417989418% + 41px)', // Keeps the aspect ratio and additional padding
+    height: 0,
+    width: '100%'}}>
+    <iframe 
+        src="https://demo.arcade.software/bAVpkNAanVDlBTyeRwJy?embed&show_copy_link=true"
+        title=""
+        style={{
+            position: 'absolute',
+            top: 0,
+            left: 0,
+            width: '100%',
+            height: '100%',
+            colorScheme: 'light'
+        }}
+        frameborder="0"
+        loading="lazy"
+        webkitAllowFullScreen
+        mozAllowFullScreen
+        allowFullScreen
+        allow="clipboard-write">
+    </iframe>
+</div>
+<p></p>
+
+:::info
+After switching to an environment, FlutterFlow generates code specific to that environment, for any of the following interactions:
+- Test / Run mode sessions
+- Local Run
+- Code export
+- Deployment
+:::
+
+### Configuring Firebase for each Environment
+
+After creating and switching to a new environment, you'll need to [**manually configure a new Firebase project**](../../../ff-integrations/firebase/connect-to-firebase-setup.md#connect-an-existing-firebase-project-manually) for that environment. This requires setting up a new project in the Firebase console and linking it to the selected environment in FlutterFlow. 
+
+:::info
+- You must complete the Firebase setup for the new environment before fully testing it. However, this doesn't stop you from continuing to run and test your app in other environments. Just switch back to production, and you can keep testing while finishing the setup for the new environment.
```

Comment:
> Just switch back to production, and you can keep testing while finishing the setup for the new environment.

Do we want to say that? Ideally we shouldnt encourage them to use Production env for testing right?